File: コーディングによる問題解決/03 アルゴリズム 初級/全探索/02_整数Vが何個あるか/2.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const N = Number(input[0]);
const V = Number(input[1]);
const A = input.slice(2).map(Number);

let cnt = 0;
for (let i = 0; i < N; i++){
    if (A[i] == V){
        cnt++;
    }
}

console.log(cnt);

File: Coding/03_Beginner/Array/Python/2.py
Q = int(input())
Array = [3,1,4,1,5,9,2,6,5,3]

for i in range(Q):
    query = list(map(int,input().split()))
    k = query[1]
    if query[0] == 0:
        print(Array[k])
    else:
        v = query[2]
        Array[k] = v